### ISSUE TITLE
Missing descriptions for -ServiceAppointmentEventsFromEmailEnabled and -WorkspaceUserEnabled parameters

### ISSUE THREAD

**Issue by emromano** [MicrosoftDocs/office-docs-powershell#10113]
Missing descriptions for **-ServiceAppointmentEventsFromEmailEnabled** and **-WorkspaceUserEnabled** parameters.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 9153892d-ff85-6117-7566-3ce55fff57c9
* Version Independent ID: 463efb5b-6bb0-97eb-bd51-f2e12cefc703
* Content: [Set-MailboxCalendarConfiguration (ExchangePowerShell)](https://learn.microsoft.com/en-us/powershell/module/exchange/set-mailboxcalendarconfiguration?view=exchange-ps)
* Content Source: [exchange/exchange-ps/exchange/Set-MailboxCalendarConfiguration.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/main/exchange/exchange-ps/exchange/Set-MailboxCalendarConfiguration.md)
* Service: **exchange-powershell**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by chrisda** [comment]
@emromano, empty parameter descriptions are a known commodity. Parameter availability vs. functionality are two different things. See [here](https://github.com/MicrosoftDocs/office-docs-powershell/issues/7429#issuecomment-808564486) for a detailed explanation.

So, are you actually trying to use those parameters (perhaps were you directed to do so by other documentation?), or are you simply pointing out that the descriptions are blank?

**Comment by emromano** [comment]
@chrisda thank you for the clarification, really appreciated.
As you said, I was just pointing out the missing description :)

To be honest, I saw the description for the parameter -workspaceuserenabled as "Fill WorkspaceUserEnabled Description" and thought that someone forgot to fill it (also my customer thought the same).

For info: I'm helping a customer implementing workspace booking but I'm not using the "WorkspaceUserEnabled" parameter, as it's not mentioned in any documentation.

The other parameter with the missing description was added to the issue for the same reason.
